app/assets/javascripts/comments.js

```diff
@@ -30,14 +30,14 @@ function init_comments_page() {
     });
     // end of removing comments
 
-
-    // comments autoload
-    var comment_page = 1;
-    load_comments();
-    // end of comments autoload
+    init_infinity_scroll(1, comments_source_url());
 
     // Helpers
 
+    function comments_source_url() {
+        return window.location.href + '/comments'
```

Comment:
I've implemented this gem for our project. But I think that in this place better to use this function, because comments is polymorphic and it will be very difficult to get proper route by this gem, if we will comments in some other place